README.md

```diff
@@ -1,54 +1,91 @@
 
-Controller actions are responsible for determining what the response is - responses are built by views.
+```touch app/controllers/static_controller.rb```
 
-For some reason we'll discuss briefly, rails is looking for a view in a very specific place. lets just make it happy and build our view in views/static/about.html.erb.
+This will create a blank controller file that we can use to map to the routing file. Since there are a number of methods built into the Rails controller system, you will also want the controller to inherit from the application controller. The new file should have code that looks like this:
 
-Let's just put some basic HTML there.
+```ruby
+class StaticController < ApplicationController
+end
+```
 
-Okay, now everything works.
+The standard naming convention for controllers is the name of the controller followed by the word ```controller```.
 
-Let's make this rendering process of choosing a view a little more explicit using the rails render method (equiv to sinatra's erb method if you're familair with it).
+If you refresh the browser now you will see a new error: ```The action 'about' could not be found for StaticController```
 
-Render can accept a string that points to a view template relative to 'views', so we can use render 'static/about'. If you think about it, that's a great place to put a view that relates to the about action in the static controller. In fact, it's such a reasonable ad convineinent place to put it that rails actually assumes, given no other render instructions, that's where you want to put it. This is a big principal in rails, convention over configuration. That's why our action worked without a render call. As a beginner we believe being explicit is easier at first then being implicit, learn the conventions and then use them.
+We're making good progress (even though we're using EDD - error driven development), and it's good to see each of the errors so that when you encounter these in your real world projects you will know how to fix them. This current error is fixed by adding the following method in the static controller:
 
-And like that we've created our first rails request - it's a static request, no data is being used, no matter what, everytime we load our about page it will be exactly the same. there is nothing dynamic about it. But that teaches us the general methodology of responding to a request with rails.
+```ruby
+def about
+end
+```
 
-draw a route.
-map it to a controller action.
-program the action
-create a template to render
-render that template.
+Hitting refresh in the browser will give you a 'Template is missing' error, specifically it says: ```Missing template static/about...```
+
+We're very close to getting our view to show up, but before we make our final addition it's important to understand the difference between explicit and implicit rendering for the views. Rails gives us two options for how views are mapped between the controller and view files:
```

Comment:
I would switch the order of the sentences "it's important to understand ..." and "Rails give us two options...".  Much nicer to set up what you're about to talk about and give context, then give some specifics (with terminology!)